### DISCUSSION TITLE
Cannot get button LEDs to stay lit (v 1.10.0)

### DISCUSSION THREAD

**Discussion by dpakman** [maartentamboer/X-Touch-Mini-FS2020#121] [Q&A] [answered]
I have installed my X-Touch Mini with MSFS2020 and have everything working with a Citation Longitude. I am using Maartentamboer’s X-Touch-Mini-FS2020 script v 1.10.0 and have been able to modify them as I need. I am also using Mobiflight to be able to control the zoom on the G3000/G5000 in the plane (awesome!).

The only issue I am having is the LED lights on the X-touch light momentarily when pressed, but do not mimic the LED state in the plane (such as AP, ALT, etc.) Seems others might be having the same problem (see the Github Issues section).

Has anyone found a workaround for this?

**Comment by dpakman** [comment, marked as answer]
Never mind…it was my fault. The global MIDI channel on the x-touch was set to 2. Needs to be 1. Sorry!

**Reply by cupax1000** [reply]
Yes! Thank you so much! It was driving me crazy.
